src/testsuite/arith128_test_i128.c

```diff
@@ -4329,6 +4329,2840 @@ test_time_i128 (void)
   printf ("\n%s mulluq end", __FUNCTION__);
   printf ("\n%s mulluq  tb delta = %lu, sec = %10.6g\n", __FUNCTION__, t_delta,
 	  delta_sec);
+
+  printf ("\n%s muludq start, ...\n", __FUNCTION__);
+  t_start = __builtin_ppc_get_timebase ();
+  for (i = 0; i < TIME_10_ITERATION; i++)
+    {
+      rc += timed_muludq ();
+    }
+  t_end = __builtin_ppc_get_timebase ();
+  t_delta = t_end - t_start;
+  delta_sec = TimeDeltaSec (t_delta);
+
+  printf ("\n%s muludq end", __FUNCTION__);
+  printf ("\n%s muludq  tb delta = %lu, sec = %10.6g\n", __FUNCTION__, t_delta,
+	  delta_sec);
+
+  printf ("\n%s muludqx start, ...\n", __FUNCTION__);
+  t_start = __builtin_ppc_get_timebase ();
+  for (i = 0; i < TIME_10_ITERATION; i++)
+    {
+      rc += timed_muludqx ();
+    }
+  t_end = __builtin_ppc_get_timebase ();
+  t_delta = t_end - t_start;
+  delta_sec = TimeDeltaSec (t_delta);
+
+  printf ("\n%s muludqx end", __FUNCTION__);
+  printf ("\n%s muludqx  tb delta = %lu, sec = %10.6g\n", __FUNCTION__, t_delta,
+	  delta_sec);
+  return (rc);
+}
+
+//#define __DEBUG_PRINT__ 1
+int
+test_subcuq (void)
+{
+  vui32_t i, j, k;
+  vui32_t e;
+  int rc = 0;
+
+  printf ("\ntest_subuq Vector subtract & Write Carry unsigned quadword\n");
```

Comment:
Wondering out loud whether it would be easier to maintain (and copy/paste) if these references to the function name were replaced by using `__FUNCTION__`?  Here and lots of other places.